src/utils/capitalized.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ['fn::capitalized']: (value) => value.charAt(0).toUpperCase() + value.slice(1).toLowerCase(),
```

Comment:
Does capitalize also enforce lowercasing the 1-end string? Might be good to err on the side of caution and not add it unless other languages/frameworks do the same.